src/main/java/it/gov/pagopa/admissibility/repository/CustomSequenceGeneratorOpsRepository.java

```diff
@@ -0,0 +1,7 @@
+package it.gov.pagopa.admissibility.repository;
+
+import reactor.core.publisher.Mono;
+
+public interface CustomSequenceGeneratorOpsRepository {
+    Mono<Long> generateSequence(String sequenceId);
```

Comment:
let's use a more specific name for the operation perfomed :)
```suggestion
    Mono<Long> nextValue(String sequenceId);
```